src/main/java/engine/config/reader/ConfigReader.java

```diff
@@ -0,0 +1,27 @@
+package engine.config.reader;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class ConfigReader {
+    public static String getProperty(PropertyKeys key) {
+        var properties = new Properties();
+        var configPath = "src/main/java/resources/config.properties";
```

Comment:
maybe there's a nicer way to create input stream from resource file? 
I think there is something like `this.getClass().getResourceAsStream("/path")` that doesn't require providing full path 🤔

Reply:
@dariabajda As per @Fishgame1's request, we will push this as it is for now. There's been another task created for adding a singleton, this will be changed as part of this other task :)